pages.fr/common/androguard.md

```diff
@@ -0,0 +1,16 @@
+# androguard
+
+> Reverse engineering tool for Android applications. Written in Python.
```

Comment:
I'm not an expert, but this doesn't sound very French to me.

Reply:
done 👍